src/test/ui/issues/issue-69225-SCEVAddExpr-wrap-flag.rs

```diff
@@ -1,5 +1,5 @@
 // run-fail
-// compile-flags: -C opt-level=3
+// compile-flags: -C opt-level=3 -Ccodegen-units=2
```

Comment:
Workaround for #81946

Reply:
I would prefer `// no-system-llvm` (can it be combined with min-llvm-version to make sure that if system llvm is >=11, we run this test anyway?) as the workaround here.

Reply:
Those exclusions combine as OR, not AND. So we can set `min-llvm-version >= 11`, of course that means we lose some coverage, but it already was broken on vanilla 9 anyway.

Reply:
Added the min version.